smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/rules/ContentPublishRule.java

```diff
@@ -69,7 +69,8 @@ public class ContentPublishRule extends ExternalResource {
     private static final Logger log = LoggerFactory.getLogger(ContentPublishRule.class);
     
     protected static final long TIMEOUT = TimeUnit.MINUTES.toMillis(5);
-    protected static final long TIMEOUT_PER_TRY = TimeUnit.MINUTES.toMillis(1);
+    // Let the check page test run for 20 minutes
+    protected static final long TIMEOUT_PER_TRY = TimeUnit.MINUTES.toMillis(20);
```

Comment:
20 minutes sounds pretty long

It would work in a pipeline - there test timeout is 30 minutes (for the whole product test module)

Though on Ecp/Evergreen we are more aggressive and have smaller timeouts.
In that case we would not get back an error to the developer but just a "module smoke timeout" or similar

Reply:
Could we do 10 minutes?

Reply:
@pat-lego, @cschneider , I agree, that 20 minutes are rather too long. The question is how long do you expect the replication to take at most until you consider it a fail? 

I don't know replication very well, so cannot tell for sure, but I'm fine with 10 minutes max,  but the shorter the better.

Reply:
10 minutes sounds good to me.